.github/ISSUE_TEMPLATE.md

```diff
@@ -5,10 +5,31 @@ Thank you for opening an issue. Please note that we try to keep the Terraform is
 ### Terraform Version
 Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
 
+This command will also output the provider version, please include that as well.
+
+### Nomad Version
+Run `nomad -v` in your target node to view which version of Nomad is running. Make sure to include the entire version output.
```

Comment:
`nomad -v` gives the version of the Nomad executable (CLI) that you invoke. 

`nomad server members` might be better here.